### PR TITLE
Add option to change value widget

### PR DESCRIPTION
**Pull request by DanMossa**
## Solution description

I wanted to change the value of the max value to an Icon. This Pr will allow you to change the label to be whatever you waet.

Let me know if i'm missing anything!

### CODE REVIEW

File: lib/src/fields/form_builder_range_slider.dart [hunk 1]
```diff
@@ -97,6 +97,11 @@ class FormBuilderRangeSlider extends FormBuilderField<RangeValues> {
   /// inform users what the currently selected value is with more context.
   final SemanticFormatterCallback? semanticFormatterCallback;
 
+  /// An alternative to displaying the text value of the slider.
+  final Widget? minValueWidget;
+  final Widget? valueWidget;
+  final Widget? maxValueWidget;
```
Comment (deandreamatias) [inline lib/src/fields/form_builder_range_slider.dart:103]: Please add a little explanation for each property

File: lib/src/fields/form_builder_range_slider.dart [hunk 2]
```diff
@@ -170,24 +178,27 @@ class FormBuilderRangeSlider extends FormBuilderField<RangeValues> {
                     children: <Widget>[
                       if (displayValues != DisplayValues.none &&
                           displayValues != DisplayValues.current)
-                        Text(
-                          effectiveNumberFormat.format(min),
-                          style: minTextStyle ?? textStyle,
-                        ),
+                        minValueWidget ??
```
Comment (deandreamatias) [inline lib/src/fields/form_builder_range_slider.dart:181]: I think that all new properties will be a function like `Widget Function(String)? minValueWidget` and here apply `minValueWidget?.call(effectiveNumberFormat.format(min))`.
With this way can access to format values in external widget. What do you think?

**Review by deandreamatias — CHANGES_REQUESTED** [review]

File: lib/src/fields/form_builder_range_slider.dart (hunk 2 above)
Comment (DanMossa) [reply lib/src/fields/form_builder_range_slider.dart:181]: I'm a little confused about what you mean.

Let's say the max value is 100, then the number under the right side of the slider will say 100.

I want to change that text widget to be something other than a text widget saying "100".

I want it to be an Icon widget.

Comment (deandreamatias) [reply lib/src/fields/form_builder_range_slider.dart:181]: Aaah ok, I understand what you mean. I thought that you will be show the `min` value inside the widget.
Either way, even you don't use the min/max value, will be nice have this value available to build the custom widget.
For example: you can create a Text with icon and the text value will be a min/max, like this:

```dart
min: 100
minValueWidget: (min) => Row(children:[Text(min), Icon(Icons.max)]) /// This min will be 100
```
With this way you can reuse the min value without duplicate again

Comment (DanMossa) [reply lib/src/fields/form_builder_range_slider.dart]: That sounds good! Let me know if there's anything else!

**Comment by deandreamatias** [comment]
I see that pipeline fails. Is because format file is wrong.
Please fix
Thanks

**Comment by deandreamatias** [comment]
Please apply `flutter format --fix` to format file
Thanks

**Comment by deandreamatias** [comment]
Hi @DanMossa
Do you can apply the changes to merge this PR?
Thanks!

**Comment by deandreamatias** [comment]
Thanks for contribution but because don't updated the branch to resolve pipeline errors, I moved this PR to #1224

**Pull request closed without merging**